pkg/controller/controller.go

```diff
@@ -322,6 +367,15 @@ func (ctrl *resizeController) resizePVC(pvc *v1.PersistentVolumeClaim, pv *v1.Pe
 		pvc = updatedPVC
 	}
 
+	// if pvc previously failed to expand because it can't be expanded when in-use
+	// we must not try expansion here
+	if ctrl.usedPVCs.hasInUseErrors(pvc) && ctrl.usedPVCs.checkForUse(pvc) {
+		// Record an event to indicate that resizer is not expanding the pvc
+		msg := fmt.Sprintf("Unable to expand %s because CSI driver %s only supports offline expansion and volume is currently in-use", pv.Name, ctrl.resizer.Name())
```

Comment:
I think print out namespace/name of the PVC is better than PV name in the event.